setup.py

```diff
@@ -25,6 +25,6 @@
             "mailbagit-gui=mailbag:gui"
         ]
     },
-    install_requires=["bagit>=1.8.1,<2", "gooey>=1.0.8.1,<2", "jsonmodels>=2.2,<=2.5.0", "extract_msg>=0.23.4,=<0.28.6"],
+    install_requires=["bagit>=1.8.1,<2", "gooey>=1.0.8.1,<2", "jsonmodels>=2.2,<=2.5.0", "extract_msg>=0.23.4"],
```

Comment:
I know this installs now, but it's not equivalent in meaning to the previous constraints - I think it would be better to change this to:

```python
"extract_msg>=0.23.4,<=0.28.6"
```

It's important to specify the upper bound because we don't want the required software to be able to go past a major version revision (and thus potentially breaking changes) without developers intending to change it and adjusting the calling code.

Reply:
Ah, I though this was a remnant from something that was removed. Upper bounds are definitely good! I have to look at the error I got, it might have just been a syntax issue.